executor.go

```diff
@@ -446,6 +445,10 @@ func runArgo(ctx context.Context, workflow *wfv1.Workflow, sink string, pipeline
 		}
 
 		if workflow.Status.Phase.Completed() {
+			
+			mixpanelClient := InitMixpanelClient("4c09914a48f08de1dbe3dc4dd2dcf90d")
```

Comment:
I don't think we should have init code inside of this loop check, just the event code. Also the SetPeople call should be handled in init, not in the event itself. This should all be wrapped and errors handled correctly in case there is no network connection.